quicklog/src/macros.rs

```diff
@@ -100,9 +105,13 @@ macro_rules! try_log {
     if $crate::is_level_enabled!($lvl) {
       use $crate::{Log, make_container};
 
-      let log_line = $crate::lazy_format::lazy_format!("[{}]\t{}", $lvl, $static_str);
-
-      $crate::logger().log(make_container!(log_line))
+      if $crate::macros::is_module_path_enabled() {
+        let log_line = $crate::lazy_format::lazy_format!("[{}][{}]\t{}", $lvl, module_path!(), $static_str);
+        $crate::logger().log(make_container!(log_line))
+      } else {
+        let log_line = $crate::lazy_format::lazy_format!("[{}]\t{}", $lvl, $static_str);
```

Comment:
Repeating these lines here is somewhat ugly, but as log_line will have different types here (lazy_format returns a closure, and all closures have unique types), this is a bit difficult to avoid.
Swapping out the string literal (either `"[{}]\t{}"` or `"[{}][{}]\t{}"`) at compile time would work, but the number of arguments to lazy_format is variadic, so swapping this out at compile time too becomes a bit messy.

Reply:
We can probably just add in the `module_path!()` in all cases 🙂 So no need for the feature flag